infinitered/ReactNativeEssentials
# Adds API service.

### Summary

- Adds API endpoints.
- Populates example List screen.
- Updates Details screen with data / tweaks styles.
- Updates Card props / tweaks styles.

### How to verify

1. `yarn`
2. `yarn start`
3. `i` & `a`
4. Go to list screen - observe.
5. Go to details screen - observe.

### Proofs

| iOS                                             | Android                                             |
| ----------------------------------------------- | --------------------------------------------------- |
| <video src="https://github.com/infinitered/ReactNativeEssentials/assets/1775841/ced91104-8fae-4efa-a049-9d65f2bb9fb2" /> | <video src="https://github.com/infinitered/ReactNativeEssentials/assets/1775841/e46d010e-5860-402d-8607-137cc508f426"/> |
